beetsplug/fetchart.py

```diff
@@ -708,8 +728,8 @@ def __init__(self):
             'enforce_ratio': False,
             'cautious': False,
             'cover_names': ['cover', 'front', 'art', 'album', 'folder'],
-            'sources': ['filesystem',
-                        'coverart', 'itunes', 'amazon', 'albumart'],
+            'sources': ['filesystem', 'coverart', 'itunes', 'amazon',
+                        'albumart', 'coverartreleasegroup'],
```

Comment:
This is really bikeshedding, but perhaps a more readable (and shorter) name would be nicer. For example, `coverart_rg` might emphasize that it's an alternative to `coverart` without getting too long about it.